### PR TITLE
Request PID 8787 & 8788

### PR DESCRIPTION
**Pull request by ncumvmclab**
We've changed our license description to open source hardware and software style.
Thank you very much!!
Contact us: ncumvmclab@gmail.com

### CODE REVIEW

File: 1209/8787/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: MVMC-lab_M128_V3
+owner: MVMC-lab
```
Comment (tannewt) [inline 1209/8787/index.md:4]: You need one folder in `org` that is named `MVMC-lab` for your organization.

**Review by tannewt — CHANGES_REQUESTED** [review]

**Review by tannewt — CHANGES_REQUESTED** [review]
Please delete the other two org files.

File: 1209/8787/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+license: NCU-MVMC-lab
+site: https://gitlab.com/MVMC-lab
+source: https://gitlab.com/MVMC-lab
```
Comment (tannewt) [inline 1209/8787/index.md:7]: Please link to the specific repository for the hardware. It should link to the software.

File: 1209/8787/index.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: MVMC-lab_M128_V3
+owner: MVMC-lab
+license: NCU-MVMC-lab
```
Comment (tannewt) [inline 1209/8787/index.md:5]: Please use a standard license: https://www.oshwa.org/sharing-best-practices/

File: 1209/8788/index.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: MVMC-lab_M3_V1
+owner: MVMC-lab
+license: NCU-MVMC-lab
+site: https://gitlab.com/MVMC-lab
+source: https://gitlab.com/MVMC-lab
```
Comment (tannewt) [inline 1209/8788/index.md:7]: Specific link here too please.

**Review by tannewt — CHANGES_REQUESTED** [review]
Thank you for removing the log orgs. Just a couple other requests.

File: 1209/8787/index.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: MVMC-lab_M128_V3
+owner: MVMC-lab
+license: MIT License
+site: https://gitlab.com/MVMC-lab
+source: https://gitlab.com/MVMC-lab/usb_transmit_program_system
```
Comment (tannewt) [inline 1209/8787/index.md:7]: This looks like the link for the software. What is the link for the specific hardware info?

Comment (ncumvmclab) [reply 1209/8787/index.md]: I've changed to hardware PCB layout project link.
It contains software link in Readme.md file in hardware project link.

Comment (tannewt) [reply 1209/8787/index.md]: Thank you for the link update. Please also include licenses in the repos.

**Comment by ncumvmclab** [comment]
Is there anything needs to fix?

File: 1209/8787/index.md (hunk 5 above)
Comment (tannewt) [reply 1209/8787/index.md]: Yes, the repos need the license as well. GitLab still shows them as all rights reserved.

Why do both PIDs have the same source repo? Is it the same hardware?

Comment (ncumvmclab) [reply 1209/8787/index.md]: They base on the similar architecture but not the same boards.
I've update them to 2 repo, both of them have software links in Readme. (Also license

Thank you!

Comment (ncumvmclab) [reply 1209/8787/index.md]: > Yes, the repos need the license as well. GitLab still shows them as all rights reserved.
> 
> Why do both PIDs have the same source repo? Is it the same hardware?

They base on the similar architecture but not the same boards.
I've update them to 2 repo, both of them have software links in Readme. (Also license

Thank you!

Comment (tannewt) [reply 1209/8787/index.md]: The two repo links look good. They still need `LICENSE` files in them though. They shouldn't have this `All rights reserved` portion on them.

Screen Shot 2021-01-09 at 5 55 37 PM

Comment (ncumvmclab) [reply 1209/8787/index.md]: Licenses in both projects were updated!

Comment (ncumvmclab) [reply 1209/8787/index.md]: > The two repo links look good. They still need `LICENSE` files in them though. They shouldn't have this `All rights reserved` portion on them.
> 
> Screen Shot 2021-01-09 at 5 55 37 PM

Licenses in both projects were updated!

Comment (ncumvmclab) [reply 1209/8787/index.md]: `LICENSEs` in both projects were updated!

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**